File: platform/sdk/nodejs/examples/graph/graph-021_proj_degree_out_2.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projDegreeOut2("hub-node");
if (!result.ok || result.outDegree !== 2) process.exit(1);
console.log("XJ-OK: graph-021_proj_degree_out_2");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-021_proj_degree_out_2.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projDegreeOut2("hub-node")
if not result["ok"] or result["outDegree"] != 2:
    sys.exit(1)
print("XJ-OK: graph-021_proj_degree_out_2")
sys.exit(0)
